backend/baseclass.pm

```diff
@@ -401,9 +404,13 @@ sub enqueue_screenshot {
         $self->write_img($image, $filename) || die "write $filename";
         $self->last_image($image);
         $self->_last_screenshot_name($filename);
-        no autodie 'unlink';
-        unlink($lastlink);
-        symlink(basename($self->_last_screenshot_name), $lastlink);
+
+        if ($want_png) {
```

Comment:
I would move this functionality into the videnencoder and only write out a png if live_log exists.